#!/usr/bin/env python
PACKAGE = "imu_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()
gen.add("threshold", double_t, 0, "A double parameter",60, 0,200)
gen.add("detector_id", int_t, 0, "detector_id", 0, 0,10)
gen.add("window_size", int_t, 0, "window_size", 20, 2,100)
gen.add("weight", double_t, 0, "sensor_weight", 1, 1,10)
gen.add("reset",   bool_t,   0, "Reset Variable",  True)
gen.add("is_disable",   bool_t,   0, "Disable Sensor",  False)

exit(gen.generate(PACKAGE, "imu_ros", "imu"))
j
